.github/scripts/get_docs_version.py

```diff
@@ -0,0 +1,40 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------------------------------
+
+# This script parses docs version from PR and set the parsed version to
+# environment variables, DOCS_CHANNEL.
+
+# We set the environment variable REL_CHANNEL based on the kind of build. This is used for
+# versioning of our assets.
+#
+# DOCS_CHANNEL is:
+# 'edge': for most builds
+# '1.0': for PR builds
+
+# DOCS_CHANNEL is used to upload assets to different paths
+
+import os
+import re
+import sys
+
+gitRefName = os.getenv("GITHUB_REF_NAME")
+
+with open(os.getenv("GITHUB_ENV"), "a") as githubEnv:
+    version = "DOCS_CHANNEL=edge"
+
+    if gitRefName is None:
+        print("This is not running in github, GITHUB_REF_NAME is null. Assuming a local build. Setting DOCS_CHANNEL to 'edge'")
+        version = "DOCS_CHANNEL=edge"
+    elif gitRefName == 'edge':
```

Comment:
Should this be case-sensitive for any case?

Reply:
The format we expect is `edge`, which should match `GIT_BASE_REF`. There shouldn't be any cases where the cases differ as far as I know

Reply:
perhaps making it case insensitive may proof this against what-ifs?

```suggestion
    elif gitRefName.lower() == 'edge':
```